### ISSUE TITLE
Bug Report: No Audio Output from FL Chatterbox TTS + PEFT Warning Despite Being Installed

### ISSUE THREAD

**Issue by xyzConfig** [filliptm/ComfyUI_Fill-ChatterBox#23]
Hi, thank you for this amazing node. I want to submit an error i encountered.

Issue Summary.
When running the FL Chatterbox TTS node in ComfyUI, no audio is generated. Both PreviewAudio and SaveAudio show 0:00 duration. The waveform output looks like this:

{ 'waveform': tensor([[0.], [0.]]), 'sample_rate': 16000 }

Additionally, I get this warning in the console:

diffusers\models\lora.py:393: FutureWarning: `LoRACompatibleLinear` is deprecated and will be removed in version 1.0.0.
Use of `LoRACompatibleLinear` is deprecated. Please switch to PEFT backend by installing PEFT: `pip install peft`.

However, I have already installed peft inside the correct virtual environment (venv_chatterbox) and restarted ComfyUI. The warning still appears, and no sound is generated.

System Info:
OS: Windows 11
Python: 3.10.11 (inside venv_chatterbox)
PyTorch: 2.7.1+cu126
GPU: NVIDIA RTX 4060 Ti 16GB
ComfyUI Version: 0.3.43 (Windows Portable)
Audio Prompt: Valid .wav file (6 seconds)
Node Settings Tried: Various values for exaggeration, cfg_weight, temperature, and seed
Behavior: No crash, but final waveform is silent (flat zero output)

Steps Taken:
1. Created a clean virtual environment: venv_chatterbox
2. Installed peft using pip install peft inside the virtual environment
3. Verified that PEFT and dependencies are listed in pip list
4. Reinstalled key packages:
    a. torch
    b. transformers
    c. safetensors
    d. huggingface_hub
    e. accelerate
5. Activated venv_chatterbox before launching ComfyUI
6. Confirmed the node is importing successfully (FL Chatterbox TTS appears)
7. Tried running several valid audio prompts and different texts
8. Restarted ComfyUI multiple times to test
9. Verified that ComfyUI logs report successful execution (e.g. “Prompt executed in 14.69 seconds”), but audio output remains empty

Expected Behavior:
The node should synthesize speech using the input text and voice prompt, and generate proper waveform output. here is the text report:

Activating venv_chatterbox...
Starting ComfyUI with venv_chatterbox...
Checkpoint files will always be loaded safely.
Total VRAM 16380 MB, total RAM 65291 MB
pytorch version: 2.7.1+cu126
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4060 Ti : cudaMallocAsync
Using pytorch attention
Python version: 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
ComfyUI version: 0.3.43
ComfyUI frontend version: 1.23.4
[Prompt Server] web root: D:\AI\ComfyUI_windows_portable-snd\venv_chatterbox\lib\site-packages\comfyui_frontend_package\static

Import times for custom nodes:
   0.0 seconds: D:\AI\ComfyUI_windows_portable-snd\ComfyUI\custom_nodes\websocket_image_save.py
   2.1 seconds: D:\AI\ComfyUI_windows_portable-snd\ComfyUI\custom_nodes\ComfyUI_Fill-ChatterBox

Context impl SQLiteImpl.
Will assume non-transactional DDL.
No target revision found.
Starting server

To see the GUI go to: http://127.0.0.1:8188
got prompt
D:\AI\ComfyUI_windows_portable-snd\venv_chatterbox\lib\site-packages\diffusers\models\lora.py:393: FutureWarning: `LoRACompatibleLinear` is deprecated and will be removed in version 1.0.0. Use of `LoRACompatibleLinear` is deprecated. Please switch to PEFT backend by installing PEFT: `pip install peft`.
  deprecate("LoRACompatibleLinear", "1.0.0", deprecation_message)
input frame rate=25
loaded PerthNet (Implicit) at step 250,000
C:\Users\shubh\AppData\Local\Programs\Python\Python310\lib\contextlib.py:103: FutureWarning: `torch.backends.cuda.sdp_kernel()` is deprecated. In the future, this context manager will be removed. Please see `torch.nn.attention.sdpa_kernel()` for the new context manager, with updated signature.
  self.gen = func(*args, **kwds)
`sdpa` attention does not support `output_attentions=True` or `head_mask`. Please set your attention to `eager` if you want any of these features.
Prompt executed in 14.69 seconds

<img width="1648" height="713" alt="Image" src="https://github.com/user-attachments/assets/d0c00074-4ade-471e-a4ba-15c15fec47c8" />

<img width="1171" height="619" alt="Image" src="https://github.com/user-attachments/assets/f1c96eb7-c2fe-4c6b-a7da-184981cf735a" />

<img width="1692" height="719" alt="Image" src="https://github.com/user-attachments/assets/0a1f0006-f1f5-4b23-bb61-50d2022aeb91" />

Please advise if there is a specific fix or workaround. Thank you in advance for your help and hard work on this node.

**Comment by millerlight** [comment]
actually you can ignore the peft warning.
this quickfix worked for me:
https://github.com/filliptm/ComfyUI_Fill-ChatterBox/issues/24

**Comment by baedesigns** [comment]
> actually you can ignore the peft warning. this quickfix worked for me: [#24](https://github.com/filliptm/ComfyUI_Fill-ChatterBox/issues/24)

i tried  this quick fix, perhaps i am not doing it correctly. the error now reads that chatter box node cannot be found.
